.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: Create a release from tag
+
+permissions: {}
+
+on:
+  push:
+    tags:
+      - 'v?[0-9]+(.[0-9]+)*(.\S+)?'
+
+jobs:
+  build:
+    name: Build and store python artifacts
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+          cache: pip
+
+      - name: Install build tool
+        run: pip install --upgrade uv
+
+      - name: Build
+        run: uv build
+
+      - name: Store python distribution artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: artifacts
+          path: python3/examples/dist/
```

Comment:
not sure this is the righ path

Reply:
It's really not, I'll test this in my repo before merging